### PR TITLE
chore: improve development-env

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* reinstalled the dependencies in monorepo root
  * `typescript`
  * `jest`
  * `webpack`, `plugins`, `loader`
  * `snowpack`
  * `eslint`
* classified the config files for root config.
* updated version
  * `webpack` (`v4` => `v5`)
  * `snowpack` (`v2` => `v3`)
  * `eslint` (`v6` => `v7`)
* added `webpack serve` command for IE test
* fixed broken build config in webpack
#### TODO
* should apply the env to wrapper project and other plugins
* applied `npm workspace`
* integrated dev server

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/types/index.d.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// Type definitions for TOAST UI Editor v3.0.0
-// TypeScript Version: 4.0.2
+// Type definitions for TOAST UI Editor vundefined
```
Comment (seonim-ryu) [inline apps/editor/types/index.d.ts:1]: 버전을 못가져오나보네요..

Comment (js87zz) [reply apps/editor/types/index.d.ts:1]: 아 이건 확인해볼게요~

File: apps/editor/webpack.config.js [hunk 2]
```diff
@@ -208,10 +102,135 @@ function setProductionConfigForAll(config) {
...
+            failOnError: false,
+          }),
+        ],
+        externals: [
+          {
+            'prosemirror-commands': {
+              commonjs: 'prosemirror-commands',
+              commonjs2: 'prosemirror-commands',
+              amd: 'prosemirror-commands',
+            },
```
Comment (seonim-ryu) [inline apps/editor/webpack.config.js:186]: CDN은 프로즈미러 다 포함해서 나가는거죠??

Comment (js87zz) [reply apps/editor/webpack.config.js:186]: 넵 맞습니다~

**Review by seonim-ryu — APPROVED** [review]
랩퍼도 타입스크립트로 다 변경되었군요 👍 고생하셨습니다~

**Pull request merged**